ash/src/extensions/khr/timeline_semaphore.rs

```diff
@@ -7,22 +7,16 @@ use std::mem;
 #[derive(Clone)]
 pub struct TimelineSemaphore {
     handle: vk::Instance,
-    timeline_semaphore_fn: vk::KhrTimelineSemaphoreFn,
+    fns: vk::KhrTimelineSemaphoreFn,
 }
 
 impl TimelineSemaphore {
     pub fn new(entry: &Entry, instance: &Instance) -> Self {
-        let timeline_semaphore_fn = vk::KhrTimelineSemaphoreFn::load(|name| unsafe {
-            mem::transmute(entry.get_instance_proc_addr(instance.handle(), name.as_ptr()))
+        let handle = instance.handle();
+        let fns = vk::KhrTimelineSemaphoreFn::load(|name| unsafe {
+            mem::transmute(entry.get_instance_proc_addr(handle, name.as_ptr()))
```

Comment:
Same comment here, `TimelineSemaphore` is a good contender for getting `device`-ified. I can take care of these changes as well if you prefer :)

Reply:
#499